components/admin/requests/physician-assessment/Card.tsx

```diff
@@ -0,0 +1,139 @@
+    }
+  );
+
+  if (!physicianAssessment) {
+    return null;
+  }
+
+  const {
+    disability,
+    disabilityCertificationDate,
+    patientCondition,
+    otherPatientCondition,
+    temporaryPermitExpiry,
+    permitType,
+  } = physicianAssessment;
+  return (
+    <PermitHolderInfoCard
+      colSpan={7}
+      header={`Physician’s Assessment`}
+      updated={isUpdated}
+      divider
+      isSubsection={isSubsection}
+      editModal={false}
+    >
+      <VStack align="left" spacing="12px">
+        <Grid
+          gridRowGap="12px"
+          gridColumnGap="20px"
+          templateColumns="200px 1fr"
+          gridAutoRows="minmax(32px, auto)"
+        >
+          <GridItem>
+            <Text as="p" textStyle="body-regular" textAlign="left">
+              Medical name of disabling condition(s)
+            </Text>
+          </GridItem>
+          <GridItem>
+            <Text as="p" textStyle="body-regular" textAlign="left">
+              {disability}
+            </Text>
+          </GridItem>
+          <GridItem>
+            <Text as="p" textStyle="body-regular" textAlign="left">
+              Certification date
+            </Text>
+          </GridItem>
+          <GridItem>
+            <Text as="p" textStyle="body-regular" textAlign="left">
+              {formatDate(disabilityCertificationDate)}
+            </Text>
+          </GridItem>
+          <GridItem>
+            <Text as="p" textStyle="body-regular" textAlign="left">
+              Condition
+            </Text>
+          </GridItem>
+          <Wrap>
+            <Badge bgColor="background.informative">{titlecase(patientCondition as string)}</Badge>
+          </Wrap>
+          {patientCondition === 'OTHER' && (
+            <>
+              {/* Condition description */}
+              <GridItem>
+                <Text as="p" textStyle="body-regular" textAlign="left">
+                  Condition description
+                </Text>
+              </GridItem>
+              <GridItem>
+                <Text as="p" textStyle="body-regular" textAlign="left">
+                  {otherPatientCondition}
+                </Text>
+              </GridItem>
+            </>
+          )}
+          <GridItem>
+            <Text as="p" textStyle="body-regular" textAlign="left">
+              Impairment type
+            </Text>
+          </GridItem>
+          <GridItem>
+            <PermitTypeBadge variant={permitType || 'PERMANENT'} />
+          </GridItem>
+          {permitType === 'TEMPORARY' && (
+            <>
+              <GridItem>
+                <Text as="p" textStyle="body-regular" textAlign="left">
+                  Permit expiry date
+                </Text>
+              </GridItem>
+              <GridItem>
+                <Text as="p" textStyle="body-regular" textAlign="left">
+                  {formatDate(temporaryPermitExpiry)}
+                </Text>
+              </GridItem>
+            </>
+          )}
```

Comment:
I wanted to confirm if the permit expiry date row is only rendered when the permit type is Temporary. I'll ask designers when I ping them for a design review

Reply:
Same as above, this is my understanding too

Reply:
this is correct!